functions/system.bash

```diff
@@ -172,47 +165,49 @@ permissions_corrections() {
 
   if is_pine64; then
     cond_redirect groupadd gpio
-    cond_redirect cp $BASEDIR/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+    cond_redirect cp "$BASEDIR"/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+    # shellcheck disable=SC2016
     cond_redirect sed -i -e '$i \chown -R root:gpio /sys/class/gpio \n' /etc/rc.local
+    # shellcheck disable=SC2016
     cond_redirect sed -i -e '$i \chmod -R ug+rw /sys/class/gpio \n' /etc/rc.local
   fi
 
   for pGroup in audio bluetooth dialout gpio tty
   do
     if getent group "$pGroup" > /dev/null 2>&1 ; then
       cond_redirect adduser openhab "$pGroup"
-      cond_redirect adduser "$username" "$pGroup"
+      cond_redirect adduser "$username $pGroup"
     fi
   done
   cond_redirect adduser "$username" openhab
 
   #
   openhab_folders=(/etc/openhab2 /var/lib/openhab2 /var/log/openhab2 /usr/share/openhab2)
-  cond_redirect chown openhab:openhab /srv /srv/README.txt /opt
+  cond_redirect chown "openhab:$username" /srv /srv/README.txt
```

Comment:
why changing this?

Reply:
This goes for all permission related changes down below. Please open a separate PR.

Reply:
ok

Reply:
Not resolved. Permissions are still changed.

Reply:
which line(s) do you mean?

Reply:
which line(s) beyond 191/186 (above) do you mean ??

Reply:
Ok, I don't know what exactly went wrong but it wasn't my intention to change permissions here.
Changed this again to now only be an obvious fix for shellcheck.